firmware/application/apps/pocsag_app.hpp

```diff
@@ -65,7 +65,12 @@ class POCSAGAppView : public View {
     NavigationView& nav_;
     RxRadioState radio_state_{};
     app_settings::SettingsManager settings_{
-        "rx_pocsag", app_settings::Mode::RX};
+        "rx_pocsag",
+        app_settings::Mode::RX};
+
+    static constexpr std::string_view enable_logging = "enable_logging";
```

Comment:
Many of these parameter name strings will be the same between apps.  I'd suggest putting them all in a common module to save ROM space.